### PR TITLE
Add Azure File CSI Driver Job Configuration

### PR DESCRIPTION
**Pull request by chewong**
Fixes kubernetes-sigs/azurefile-csi-driver#62. This will obsolete #12386.

/assign @andyzhangx @feiskyer
/cc @ritazh
/area provider/azure

### CODE REVIEW

**Comment by feiskyer** [comment]
please get https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/120 merged first.

**Comment by chewong** [comment]
@feiskyer Is it possible to get this PR merged first so we can test https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/120 with Prow?

File: config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  kubernetes-sigs/azurefile-csi-driver:
+  - name: pull-azurefile-csi-driver-verify
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190927-21e0205-master
```
Comment (feiskyer) [inline config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml]: use image `gcr.io/k8s-testimages/kubekins-e2e:v20191002-1ea0701-master`

File: config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      testgrid-num-columns-recent: '30'
+  - name: pull-azurefile-csi-driver-unit
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190927-21e0205-master
```
Comment (feiskyer) [inline config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml]: same here

File: config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  - name: pull-azurefile-csi-driver-integration
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-cloudprovider-azure-cred: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190927-21e0205-master
```
Comment (feiskyer) [inline config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml]: same here

**Comment by feiskyer** [comment]
@cblecker could you have a look at the PR?

/assign @cblecker

**Pull request merged**